warrior/Framework/Utils/kafka_utils.py

```diff
@@ -0,0 +1,206 @@
+'''
+Copyright 2019, Fujitsu Network Communications, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+from Framework.Utils.print_Utils import print_error, print_exception
+try:
+    from kafka import KafkaConsumer
+    from kafka import KafkaProducer
+    from kafka import TopicPartition
+    from kafka.admin import KafkaAdminClient, NewTopic, NewPartitions
+except ImportError as err:
+    print_error("Module kafka is not installed, Refer to the exception trace below for more details")
+    print_exception(err)
+
+class WarriorKafkaConsumer(object):
+    """
+    This class contains all kafka consumer methods
+    """
+    def __init__(self, *topics, **configs):
+        """
+        Create Kafka Consumer object
+        """
+        self.kafka_consumer = KafkaConsumer(*topics, **configs)
```

Comment:
All the Exceptions need to be handled.  Consumer/Producer/Client
https://github.com/dpkp/kafka-python/blob/master/kafka/errors.py